environments/nodejs/server.js

```diff
@@ -66,6 +66,7 @@ app.use(morgan('combined'))
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 app.use(bodyParser.raw());
+app.use(bodyParser.text({ type : "*/*" }));
```

Comment:
Shouldn't we use text for test/plain and raw for */*?

Reply:
I would agree with you as it seems like a more logical setup. However with HTTP/javascript/express it is more of conforming with the user's expectation. If we use 'raw' to parse everything else, it would turn bodies of content-types like `text/html` into int-arrays. What about using "text/*" instead?

Reply:
Ok, seems reasonable to treat all `text/*` as text.